designs/unice/config/index.js

```diff
@@ -32,6 +32,16 @@ const config = {
       'gussetWidth',
       'gussetLength'
     ],
+    style: [
+      'rise',
+      'legOpening',
+      'frontDip',
+      'frontCurve',
+      'backDip',
+      'backCurve',
```

Comment:
You'll also want to make sure you add the new `frontCurve` and `backCurve` settings to i18n strings and the documentation markdown pages.

Reply:
Where do I find those?

Reply:
The i18n strings are in:
`packages/i18n/src/locales/en/options/unice.yml`

And, the docs are in this directory:
`markdown/org/docs/patterns/unice/options`
You would create `frontcurve` and `backcurve` subdirectories and an `en.md` file within each new subdirectory. (I believe that all the other language files will automatically get created.)

You can look around the options directory to see how the other options are documented. (I'm not sure how the superimposed images are created, so you might want to omit the "Effect of this option on the pattern" sections for now.)